tests-e2e/util/VideoSection.ts

```diff
@@ -0,0 +1,55 @@
+import { Page, Locator, FrameLocator, expect } from "@playwright/test"
+
+class VideoSection {
```

Comment:
**Requested Change:** Could you merge this with `VideoElement`? It got added in #1138 

Be sure to look at the existing methods. E.g., I think `VideoElement.within` would replace the `.videoSection` method here. ("within" is probably a better name)